src/OceanSeaIceModels/ocean_sea_ice_model.jl

```diff
@@ -73,6 +76,26 @@ function initialize!(model::OSIM)
     return nothing
 end
 
+initialize_jld2_file!(filepath, init, jld2_kw, including, outputs, model::OSIM) =
+    initialize_jld2_file!(filepath, init, jld2_kw, including, outputs, model.ocean.model)
+
+write_output!(c::Checkpointer, model::OSIM) = write_output!(c, model.ocean.model)
+
+function set!(sim::OSIMSIM, pickup::Union{Bool, Integer, String})
```

Comment:
@glwagner should we type this so that it only works for OSIMSIM with PrescribedAtmosphere?

Reply:
[4f5ff2e](https://github.com/CliMA/ClimaOcean.jl/pull/401/commits/4f5ff2e6ef6ae38e71ef1112316fc552205ce32f)